### PR TITLE
Pre privacy updates

### PR DESCRIPTION
**Pull request by ineilson**
Miscellaneous, non-privacy-related changes, in preparation for data privacy updates.

### CODE REVIEW

File: htdocs/web_portal/img/README.md [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
-|user.png|[Link](http://www.iconarchive.com/show/ravenna-3d-icons-by-double-j-design/Users-icon.html)|CC Attribution 4.0|Double-J Design|
-|search.png|[Link](http://www.softicons.com/business-icons/business-icons-by-petalart/search-icon)|CC Attribution-NoDerivs 3.0 Unported|	PetalArt|
-|project.png|[Link](http://www.icons101.com/icon/id_51397/setid_1789/Free_3D_Printer_by_AhaSoft/Project)|CC Attribution|Aha-Soft (website)|
 |home.png|[Link](http://iconbug.com/detail/icon/2884/blue-home-folder/)|CC Attribution 3.0 Unported|[Link](http://www.iconarchive.com/icons/oxygen-icons.org/oxygen/authors.txt)|
-|star.png|[Link](http://www.softicons.com/web-icons/web-2-icon-set-by-anders-bjarnle/star-icon)|Free Use of icon|Anders Bjarnle|
+|key.png|[Link](http://www.softicons.com/toolbar-icons/build-icons-by-design-kindle/key-icon)|Free for commercial use|	Design Kindle|
+|network.png|[Link](https://findicons.com/icon/169709/globe)|GNU Lesser General Public License|YellowIcon|
+|new_window.png|[Link](http://www.softicons.com/toolbar-icons/cyan-bitcons-icons-by-some-random-dude/new-window-icon)|CC Attribution-ShareAlike 3.0 United States|Some Random Dude (<-
+actually the author information)|
```
Comment (gregcorbett) [inline htdocs/web_portal/img/README.md:12]: I think this should be one line? It looks like the line break messes up the markdown parsing.

File: htdocs/web_portal/img/README.md [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
+|search.png|[Link](http://www.softicons.com/business-icons/business-icons-by-petalart/search-icon)|CC Attribution-NoDerivs 3.0 Unported|	PetalArt|
 |server.png|[Link](https://findicons.com/icon/185522/network_server)|GNU Lesser General Public License|silvestre herrera|
 |service.png|[Link](https://findicons.com/icon/227231/server)|GNU Lesser General Public License|David Vignoni|
-|network.png|[Link](https://findicons.com/icon/169709/globe)|GNU Lesser General Public License|YellowIcon|
-|person.png|[Link](http://www.iconarchive.com/show/ecommerce-business-icons-by-designcontest/admin-icon.html)|CC Attribution 4.0|DesignContest|
+|star.png|[Link](http://www.softicons.com/web-icons/web-2-icon-set-by-anders-bjarnle/star-icon)|Free Use of icon|Anders Bjarnle|
+|tick.png|[Link](http://www.softicons.com/system-icons/crystal-clear-icons-by-everaldo-coelho/actions-agt-action-success-icon)|GNU Lesser General Public License |Everaldo Coelho|
+|trash.png|[Link](http://www.softicons.com/business-icons/ecommerce-and-business-icons-by-designcontest.com/trash-icon)|CC Attribution 3.0 Unported|DesignContest.com|
```
Comment (gregcorbett) [inline htdocs/web_portal/img/README.md:23]: It looks like this commit also adds `trash.png` to the `README.md`? If so, could the commit message be amended to mention that?

File: htdocs/web_portal/img/README.md (hunk 1 above)
Comment (ineilson) [reply htdocs/web_portal/img/README.md:12]: Good call.

File: htdocs/web_portal/img/README.md (hunk 2 above)
Comment (ineilson) [reply htdocs/web_portal/img/README.md:23]: Will do.

**Review by gregcorbett — APPROVED** [review]
LGTM

**Pull request merged**